resilience4j-retry/src/main/java/io/github/resilience4j/retry/RetryConfig.java

```diff
@@ -150,7 +171,7 @@ public Builder(RetryConfig baseConfig) {
                 this.intervalFunction = (x) -> waitDuration.toMillis();
             } else {
                 throw new IllegalArgumentException(
-                    "waitDurationInOpenState must be a positive value");
```

Comment:
is this considered a breaking change?

I think it was copy pasted from circuit breaker 😄

Reply:
No :) we should fix the message :)